config/packages/jms_serializer.yaml

```diff
@@ -0,0 +1,13 @@
+jms_serializer:
+    visitors:
+        xml:
+            format_output: '%kernel.debug%'
+#    metadata:
+#        auto_detection: false
+#        directories:
+#            any-name:
+#                namespace_prefix: "My\\FooBundle"
+#                path: "@MyFooBundle/Resources/config/serializer"
+#            another-name:
+#                namespace_prefix: "My\\BarBundle"
+#                path: "@MyBarBundle/Resources/config/serializer"
```

Comment:
I would remove commented configuration here and below, they can be useful if we provide project for somebody to base on, this is just a Sylius demo which we maintain :)